### DISCUSSION TITLE
我的场景是需要DET提取文本行键值对

### DISCUSSION THREAD

**Discussion by zzk2021** [PaddlePaddle/PaddleOCR#14186] [Q&A]
比如     “目标函数：DBloss” 中获取键值对“目标函数”和“DBloss” ，使用det是连续检测一个文本行的，我需要分为两个文本区域。便于进行后续的ser和re训练。目前训练效果不太好，我看pp ocrv3使用了DB检测头，使用了Vatti clipping algorithm算法，我该怎么控制文本膨胀呢？
![image](https://github.com/user-attachments/assets/f616d347-7095-4e43-b535-410e7ede3b2e)
论文中的shrunk polygon有用吗？

**Comment by GreatV** [comment]
无法理解你的问题

**Reply by zzk2021** [reply]
就是一个文本行需要检测出两个文本区域，分为键合值。比如下面这种数据
![image](https://github.com/user-attachments/assets/47603e3d-a5b4-4366-8c43-d36e1d9b251b)
我们希望分开检测，如下图
![image](https://github.com/user-attachments/assets/63122413-7951-4aa0-af4f-405a090267d1)
但是det是只会检测一行，不会分开，我们标注了这样的数据进行训练，但是效果不好，请问有什么好方法吗

**Reply by GreatV** [reply]
后处理的时候根据：分割一下？

**Reply by UserWangZz** [reply]
试试PPChatOCR能不能解决你的这个需求，用大模型提取键值对

**Comment by zzk2021** [comment]
后处理效果还行，能达到要求
